README.md

```diff
@@ -255,6 +256,22 @@ services:
       - "8000:8000"
 ```
 
+The values of the environmental variables are pulled from the `.env` file.
+
+The `DENO_DEPLOYMENT_ID` variable is needed for Docker deployment of a Deno
+Fresh app for caching to work properly. Its value needs to be a token reflecting
+the build that you will be deploying. The SHA1 commit hash is a suitable token
+that can be obtained from the following command run in the repo's root folder.
+Copy its results into the value of `DENO_DEPLOYMENT_ID`. Make sure you update
+this value every time you run a new docker compose build.
+
+```sh
+# get the SHA1 commit hash of the current branch
+git rev-parse HEAD
+```
+
+Make sure this command is run and value upgraded
```

Comment:
we can prob remove this part given the re-organization of the information in the paragraph prior

Reply:
I agree that the last line (273) needs to be removed, but the `git rev-parse HEAD` command should remain as the previous paragraph points to it. Don't you agree?